src/diffusers/pipelines/aura_flow/pipeline_aura_flow.py

```diff
@@ -387,7 +387,6 @@ def __call__(
         prompt: Union[str, List[str]] = None,
         negative_prompt: Union[str, List[str]] = None,
         num_inference_steps: int = 50,
-        timesteps: List[int] = None,
```

Comment:
Even though these are non-functional but isn't this a breaking change? If so, maybe we should deprecate (with a short cycle, for sure)?

Reply:
In #10116 for Flux pipelines we replaced `timesteps` with `sigmas` but it's a good point. Let's bring in @yiyixuxu for a second opinion.

Reply:
it doesn't because the flow match scheduler does not accept `timesteps`

thanks for the fix @hlky!

Reply:
That makes sense.